hack/jenkins/e2e.sh

```diff
@@ -37,8 +37,13 @@ else
     export HOME=${WORKSPACE} # Nothing should want Jenkins $HOME
 fi
 
-# Additional parameters that are passed to ginkgo runner.
+# TODO: clean up this mess.
+# Additional parameters that are passed to the Ginkgo runner.
+GINKGO_ARGS=${GINKGO_ARGS:-"--noColor"}
+# Additional parameters that are passed to the Ginkgo e2e tests.
 GINKGO_TEST_ARGS=${GINKGO_TEST_ARGS:-""}
```

Comment:
I find GINKGO_ARGS and GINKGO_TEST_ARGS very confusing.  What's the difference?  I think that they need renaming or better documentation, as it's going to confuse not just me :-)